src/App.vue

```diff
@@ -1,9 +1,9 @@
 <template>
   <div id="app">
     <Header/>
-    <SubHeading/>
+    <SubHeading v-if="!isNominate"/>
     <router-view/>
-    <Footer/>
+    <Footer v-if="!isNominate"/>
```

Comment:
Unlike the designer page, we do have a footer, so it should be visible always
```suggestion
    <Footer>
```

Reply:
Done  .. .